fvm/context.go

```diff
@@ -118,6 +118,14 @@ func WithGasLimit(limit uint64) Option {
 	}
 }
 
+// WithSignatureVerifier sets the signature verifier
+func WithSignatureVerifier(verifier SignatureVerifier) Option {
```

Comment:
Doesn't seem like this is used. Just to check understanding, we are relying on the default signature verifier being set here https://github.com/onflow/flow-go/pull/590/files#diff-25d32325a03c8a9a47799a7764de17b16ad26a58410500d48d9afaf985648cfeR86

Reply:
We have `SignatureVerifier` in the context so I added it as a configurable. Also, made sure it's used down the lane (we were creating new one in `TransactionSignatureVerifier`